modules/ROOT/pages/_partials/predictive-query.adoc

```diff
@@ -0,0 +1,93 @@
+.Uncommited API
+NOTE: Predictive query is a xref:c-sdk::compatibility-versions-features.adoc[Uncommitted API].
+
+.Enterprise Edition only
+IMPORTANT: Predictive query is an https://www.couchbase.com/products/editions[Enterprise Edition] feature.
+
+Predictive Query enables Couchbase Lite queries to use machine learning, by providing query functions that can process document data (properties or blobs) via trained ML models.
```

Comment:
Good point, I'm never sure what to do about capitalization. It will be really good to have that covered in the style guide. Fixed in https://github.com/couchbase/docs-couchbase-lite/commit/3129b7961a85fc3ea9a2fc81280b91b367014bee